code/game/machinery/porta_turret/portable_turret.dm

```diff
@@ -219,9 +223,7 @@
 			stat &= ~NOPOWER
 			update_icon()
 		else
-			spawn(rand(0, 15))

```

Comment:
Why is this even here, just remove the delay.

Reply:
Been wondering myself, a lot of power change events use random delays fnr